Sources/ComposableArchitecture/Reducer/Reducers/BindingReducer.swift

```diff
@@ -5,7 +5,12 @@ public struct BindingReducer<State, Action>: ReducerProtocol
 where Action: BindableAction, State == Action.State {
   /// Initializes a reducer that updates bindable state when it receives binding actions.
   @inlinable
-  public init() {}
+  public init() {
+    self.init(internal: ())
+  }
+
+  @usableFromInline
+  init(internal: Void) {}
```

Comment:
So I guess with library evolution you can only call out to other `init`s when marking the `init` as `@inlinable`. So, to work around that I need to make an internal, `@usableFromInline` init, and then call out to that one from the public `init`. And on top of that, for `init`s that don't take any arguments I need to do something to disambiguate the initializers, so I passed a void value.

Please someone let me know if there's a better way.